docs/conf.py

```diff
@@ -14,14 +14,14 @@
 from pathlib import Path
 
 current_dir = Path(__file__).parents[1]
-code_directory = current_dir / "maykin_django_utils"
+code_directory = current_dir / "django_common"
```

Comment:
ah this may actually be a mistake in the default-app template!

I think what we want is to add the repository root to the `sys.path` so that you can import `maykin_common.*`, `testapp.*` and `tests.*` (the latter makes no sense to import though)

so it should probably be:

```py
repo_root = Path(__file__).parents[1]

sys.path.insert(0, str(repo_root))
```